app/controllers/albums.js

```diff
@@ -1,21 +1,26 @@
-const { requestAlbums, requestAlbumPhotos } = require('../services/typicode'),
+const { requestAlbumPhotos } = require('../services/typicode'),
   { getEmailFromToken } = require('../helpers/token'),
   User = require('../models').user,
   logger = require('../logger'),
   Purchase = require('../models').purchase,
-  { validationError, permissionError } = require('../errors');
+  { validationError, permissionError } = require('../errors'),
+  { GraphQLObjectType, GraphQLSchema, graphql } = require('graphql'),
+  { albums } = require('../graphql/albums/queries');
 
 exports.getAlbums = (req, res, next) => {
-  requestAlbums()
-    .then(json => {
-      res.status(200).send(json);
+  logger.info('Listing all albums');
+  const schema = new GraphQLSchema({ query: new GraphQLObjectType(albums) });
+
+  return graphql(schema, '{ albumsList { id, title } }')
```

Comment:
the idea is that you can wrap your endpoint in graphql, here you did exactly the opposite

Reply:
Is the new implementation better?